wp-web-push/lib/js/wp-web-push.js

```diff
@@ -215,60 +241,96 @@ if (navigator.serviceWorker) {
+  return check;
+    })();
+
+    if (!isMobile) {
+      document.querySelector('#webpush-subscription .bubble').onmouseover = function() {
+        mouseOnTooltip = true;
+        clearTimeout(hideTooltipTimeout);
+      };
+
+      document.querySelector('#webpush-subscription .bubble').onmouseout = function() {
+        mouseOnTooltip = false;
+        hideTooltip();
+      };
+
+      document.querySelector('#webpush-subscription .subscribe').onmouseover = function() {
+        mouseOnButton = true;
+        clearTimeout(hideTooltipTimeout);
+
+        localforage.setItem('button_interacted', true);
+
+        notificationsEnabled()
+          .then(function(enabled) {
+            if (enabled && Notification.permission === 'granted') {
+              setSubscriptionTip('<p>' + ServiceWorker.unsubscription_prompt + '</p><p><button class="unsubscribe default">' + ServiceWorker.unsubscription_button_text + '</button></p>', true);
+
+              document.querySelector('#webpush-subscription .unsubscribe').onclick = function() {
+                disableNotifications()
+                  .then(function() {
+                    setSubscriptionTip(ServiceWorker.unsubscribed_hint);
+                  });
+              };
+            } else {
+              setSubscriptionTip('<p>' + ServiceWorker.subscription_prompt + '</p><p><img src="' + ServiceWorker.notification_preview + '" alt="" /></p>', true);
+            }
+          });
+      };
+
+      document.querySelector('#webpush-subscription .subscribe').onmouseout = function() {
+        mouseOnButton = false;
+        hideTooltip();
+      };
+
+      document.querySelector('#webpush-subscription .subscribe').onclick = function() {
+        localforage.setItem('button_interacted', true);
+        setNotificationsIndicator(false);
+
+        notificationsEnabled()
+          .then(function(enabled) {
+            if (enabled && Notification.permission === 'granted') {
+              // Do nothing.
+            } else {
+              enableNotifications(true)
+                .then(function() {
+                  setSubscriptionTip(ServiceWorker.unsubscription_hint);
+                });
+            }
+          });
+      };
+    }
+    else {
+      var closeButton = document.querySelector('#webpush-subscription .actions .dismiss');
+
+      closeButton.innerHTML = ServiceWorker.close_button_text;
+
+      document.querySelector('#webpush-subscription .subscribe').onclick = function() {
+        localforage.setItem('button_interacted', true);
+        setNotificationsIndicator(false);
+
+        notificationsEnabled()
+          .then(function(enabled) {
+            if (enabled && Notification.permission === 'granted') {
+              showUnsubscribe();
+            } else {
+              showSubscribe();
```

Comment:
Also need to call enableNotifications(true) here, otherwise we're not subscribing the user.